atomic_reactor/util.py

```diff
@@ -633,10 +634,12 @@ def default(self):
         In such case, the v1 schema should be used when interacting
         with the registry.
         """
-        return self.v2 or self.v1
+        return self.v2_list or self.v2 or self.v1
 
 
 def get_manifest_media_type(version):
+    if version == 'list':
```

Comment:
Maybe use 'v2_list' throughout, instead of 'list'? There's another type in the works as well, 'oci', so at some point we might need to define some constants to hold these names.